hw/ip/lc_ctrl/data/lc_ctrl.hjson

```diff
@@ -5,57 +5,459 @@
+                  In order to have exclusive access to this register, SW must first claim the associated
+                  hardware mutex via !!CLAIM_TRANSITION_IF.
+                  ''',
+        count:    "NumTokenWords", // 4 x 32bit = 128bit
+        swaccess: "rw",
+        hwaccess: "hrw",
+        hwqe:     "true",
+        hwext:    "true",
+        regwen:   "TRANSITION_REGWEN",
+        cname:    "WORD",
+        fields: [
+          { bits: "31:0"
+          }
+        ]
+      }
+    },
+    { name: "TRANSITION_TARGET",
+      desc: "This register exposes the decoded life cycle state.",
+      swaccess: "rw",
+      hwaccess: "hrw",
+      hwqe:     "true",
+      hwext:    "true",
+      regwen:   "TRANSITION_REGWEN",
       fields: [
-        { bits: "1:0"
-          name: "DUMMY"
-          desc: '''foo
+        { bits: "NumLcStateBits-1:0"
+          name: "STATE"
+          desc: '''
+                Note that this register is shared with the life cycle TAP interface.
+                In order to have exclusive access to this register, SW must first claim the associated
+                hardware mutex via !!CLAIM_TRANSITION_IF.
                 '''
+          enum: [
+            { value: "0",
+              name:  "RAW",
+              desc:  "Raw lifecycle state after fabrication where all functions are disabled."
+            }
+            { value: "1",
+              name:  "TEST_UNLOCKED0",
+              desc:  "Unlocked test state where debug functions are enabled."
+            }
+            { value: "2",
+              name:  "TEST_LOCKED0",
+              desc:  "Locked test state where where all functions are disabled."
+            }
+            { value: "3",
+              name:  "TEST_UNLOCKED1",
+              desc:  "Unlocked test state where debug functions are enabled."
+            }
+            { value: "4",
+              name:  "TEST_LOCKED1",
+              desc:  "Locked test state where where all functions are disabled."
+            }
+            { value: "5",
+              name:  "TEST_UNLOCKED2",
+              desc:  "Unlocked test state where debug functions are enabled."
+            }
+            { value: "6",
+              name:  "TEST_LOCKED2",
+              desc:  "Locked test state where debug all functions are disabled."
+            }
+            { value: "7",
+              name:  "TEST_UNLOCKED3",
+              desc:  "Unlocked test state where debug functions are enabled."
+            }
+            { value: "8",
+              name:  "DEV",
+              desc:  "Development life cycle state where limited debug functionality is available."
+            }
+            { value: "9",
+              name:  "PROD",
+              desc:  "Production life cycle state."
+            }
+            { value: "10",
+              name:  "PROD_END",
+              desc:  "Production life cycle state."
```

Comment:
Is this the same desc as PROD enum?

Reply:
in principle yes. although I should also add a sentence saying that you cannot transition from PROD_END into RMA. let me add that...

Reply:
Oh I see what you mean!